uploader/uploader.go

```diff
@@ -79,41 +86,59 @@ func uploadDir(ctx context.Context, uploader Backend, local string, remote strin
 		if err != nil {
 			errs = multierror.Append(errs, err)
 		} else {
-			uploadErrs := make([]error, len(finfos))
-
-			// Iterate over each file and upload it
+			log.WithField("file", local).Info("is a directory - to read files")
```

Comment:
Do we need to make normal uploads concurrent, given that we only care about slow container shutdown, can this logic only happen in uploadAllLogFiles?

Reply:
Since the upload/uploader.go change is not used currently I would not worry about this being exercised right now. As it is, it is already concurrent but the concurrency is not limited so I am planning on leaving the change there.